todo_app/templates/index.html

```diff
@@ -12,8 +12,28 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List items in todo list -->
+        <h3>To Do:</h3>
         {% for itm in todoitems %}
```

Comment:
Personal preference, but I tend to find writing the extra letter in `item` here adds very little overhead but does make reading the code a shade easier!